tox.ini

```diff
@@ -100,10 +101,10 @@ commands =
   os.path.isdir(dist_dir) or sys.exit(0); \
   print("Removing \{!s\} contents...".format(dist_dir), file=sys.stderr); \
   shutil.rmtree(dist_dir)'
-  {envpython} -m pep517.build \
-    --source \
-    --binary \
-    --out-dir {toxinidir}/dist/ \
+  {envpython} -m build \
+    --sdist \
+    --wheel \
```

Comment:
Drop these. Starting with build v0.5.0, it builds sdist and then builds a wheel from that sdist tarball. But if you specify `--sdist` / `--wheel`, both end up being built from the Git checkout which is not how pip will behave if it hits sdist.
```suggestion
```

Reply:
Note: all the other comments except for this one are non-blockers. Apply this patch and feel free to merge.